parla/cpu.py

```diff
@@ -1,66 +1,17 @@
 import logging
 from typing import Dict, Collection
 
-# FIXME: This load of numpy causes problems if numpy is multiloaded. So this breaks using VECs with parla tasks.
-#  Loading numpy locally works for some things, but not for the array._register_array_type call.
```

Comment:
I think this is still an issue with this code. Why did you remove the FIXME.

Reply:
I put it back in. This is only partially fixed now.